docs/transactions/payment-request/create_payment_request.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Create a payment request b58 code to give to someone else
+---
+
+# Create Payment Request
+
+## Parameters
+
+| Required Param | Purpose | Requirements |
+| :--- | :--- | :--- |
+| `account_id` | The account on which to perform this action. | Account must exist in the wallet. |
+| `amount_pmob` | The amount of pMOB to send in this transaction. | `u64` |
+
+| Optional Param | Purpose | Requirements |
+| :--- | :--- | :--- |
+| `subaddress_index` | The subaddress index on the account to send the transaction | `i64` |
```

Comment:
I don't think `send` makes sense here. This is the address at which you want the recipient to pay you.